addons/api/tests/unit/services/resource-filtering-store-test.js

```diff
@@ -0,0 +1,29 @@
+import { module, test, skip } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | resource-filtering-store', function (hooks) {
+  setupTest(hooks);
+  let service, store;
+  hooks.beforeEach(function () {
+    service = this.owner.lookup('service:resource-filtering-store');
+    store = this.owner.lookup('service:store');
+
+    store.createRecord('session', {
+      id: 's_1',
+      scope: {
+        id: '123',
+      },
+    });
+  });
+
+  // TODO: Replace this with your real tests.
+  test('it exists', function (assert) {
+    assert.ok(service);
+  });
+
+  // TODO: Is it possible to capture mirage API requests in this test?
+  skip('it supports query filters', function (assert) {
```

Comment:
@randallmorey , Can I get some help on addressing how to test this new service? Is it worth hooking in mirage or is there an easier way to look at request uri?

Reply:
Should I be mocking out store service?